index.js

```diff
@@ -84,9 +83,14 @@ async function exportEvents(events, { global, storage }) {
 
     rows = rows.join().split('\n')
 
-
-    const data = await storage.get('data', [])
-
+    let data = null
+    // this is an intentional change as testing storage with posthog is a little difficult to simulate
+    if (!global.tests) {
```

Comment:
@yakkomajuri  Could you confirm if this is okay to have as this is the only way to add the test from what i can understand.

Reply:
there are utils available to mock `storage`. 

one can also mock it very easily:

```js
const storageObject = {} 

const mockedStorage = {
    get: (key, default) => storageObject[key] || default,
    set: (key, value) => { storageObject[key] = value }
```